src/main/java/com/redhat/devtools/intellij/common/utils/DownloadHelper.java

```diff
@@ -136,7 +136,7 @@ public String downloadIfRequired(String toolName, URL url) throws IOException {
                     command = ProgressManager.getInstance().run(new Task.WithResult<String, IOException>(null, "Downloading " + toolName, true) {
                         @Override
                         public String compute(@NotNull ProgressIndicator progressIndicator) throws IOException {
-                            return HttpRequests.request(platform.getUrl().toString()).connect(request -> {
+                            return HttpRequests.request(platform.getUrl().toString()).useProxy(true).connect(request -> {
```

Comment:
It looks to me that this part of the code doesn't change the current behavior as the `useProxy` method set the `myUseProxy` variable to true but the default value of `myUseProxy` is already set to true, so this is only used to improve readibility, right?

Reply:
I didn't catch that `useProxy` was the default but I think it's safer to enforce it in case they decide to change the default